tasks/bash.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 # Delegate to facter if available
+export PATH=$PATH:"/opt/puppetlabs/bin/"
```

Comment:
Should probably put `$PATH` in the quotes, i.e. `PATH="$PATH:/opt/puppetlabs/bin"`. The new path doesn't have spaces, but old one might.

Reply:
I am wondering if modifying `$PATH` is the best option. On my laptop, when I append `/opt/puppetlabs/bin` to `$PATH` an old version of facter that does not support `--show-legacy` is used (2.5.1 from `~/.rbenv/shims`). When I prepend the newer version from `opt/puppetlabs/bin` is used (3.11.3). It seems like if you prepend to the path you override existing precedence. Is a case like this important?

Reply:
Your laptop isn't like most environments will be setup. This will require Facter 3. I still think we should defer to PATH if someone has configured something different.